WordPress/gutenberg
# Add remaining editable shortcuts

## Description

See #71.

This PR adds the remaining shortcuts for editable:

* `access+d` for strikethrough (`<del>` tag).
* `access+c` for code.
* `access+a` and `meta+k` to add a link.
* `access+s` to remove a link.

`access` means `ctrl+alt` on Mac and `shift+alt` on Windows.
`meta` means `cmd` on Mac and `ctrl` on Windows.

## How Has This Been Tested?

1. Press `access+d` and the text should be struck through. Press it again and the formatting should be removed.
2. Press `access+x` and the text should be code. Press it again and the formatting should be removed.
3. Press `access+a` and the pop up to add a link should appear focussed.
4. Press `meta+k` and the pop up to add a link should appear focussed.
5. Press `access+s` after putting the caret on a link. The link should be removed.

## Checklist:
